packages/access-om3-nuopc/package.py

```diff
@@ -56,9 +57,15 @@ class AccessOm3Nuopc(CMakePackage):
     depends_on("mpi")
     depends_on("netcdf-fortran@4.6.0:")
     depends_on("esmf@8.3.0:")
-    depends_on("fms@2021.03:")
-    depends_on("parallelio@2.5.10:")
     depends_on("fortranxml@4.1.2:")
+    depends_on("fms@2021.03: build_type==RelWithDebInfo precision=64 +large_file ~gfs_phys ~quad_precision ~openmp", when="~openmp")
+    depends_on("fms@2021.03: build_type==RelWithDebInfo precision=64 +large_file ~gfs_phys ~quad_precision +openmp", when="+openmp")
```

Comment:
Looking at the spack docs, it seems one could write this in the following way:
```suggestion
    depends_on("fms@2021.03: build_type==RelWithDebInfo precision=64 +large_file ~gfs_phys ~quad_precision)
    depends_on("fms +openmp", when="+openmp")
    depends_on("fms ~openmp", when="~openmp")
```
I think this reads much better.